tests/migration/online_migration/pre_migration.pm

```diff
@@ -48,6 +48,9 @@ sub remove_kgraft_patch {
 sub run {
     select_console 'root-console';
 
+    # set_zypp_single_rpmtrans
```

Comment:
This make me confused, you have run the function in zypper_patch why still need to run it here?

Reply:
Besides, Copyright 2016 -> Copyright 2016-2022

Reply:
now use the setenv and there's reboot after zypper_patch. for copyright please see:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md?plain=1#L50